src/Game/Maze/Image.java

```diff
@@ -0,0 +1,52 @@
+package Game.Maze;
+
+
+import javax.imageio.ImageIO;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.WindowConstants;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+public class Image {
+
+    public static void main(String[] args) throws IOException {
+        JFrame frame = buildFrame();
+
+        final BufferedImage edgebottomleft = ImageIO.read(new File("d:\\_maze\\resources\\Wall\\wall_edge_bottom_left.pngl"));
+        final BufferedImage edgebottomright = ImageIO.read(new File("D:\\_maze\\resources\\Wall\\wall_edge_bottom_right.png"));
+        final BufferedImage edgetopleft = ImageIO.read(new File("D:\\_maze\resources\\Wall\\wall_edge_top_left.png"));
+        final BufferedImage edgetopright = ImageIO.read(new File("d:\\_maze\resources\\Wall\\wall_edge_top_right.png"));
+        final BufferedImage edgeleft = ImageIO.read(new File("d:\\_maze\\resources\\Wall\\wall_edge_left.png"));
+        final BufferedImage edgeright = ImageIO.read(new File("d:\\_maze\resources\\Wall\\wall_edge_right.png"));

```

Comment:
Chỗ này không cần hardcode ra làm gì, cứ để là `String` thôi.

```suggestion
        final String edgebottomleft = "Wall/wall_edge_bottom_left.png";
        final String edgebottomright = "Wall/wall_edge_bottom_right.png";
        final String edgetopleft = "Wall/wall_edge_top_left.png";
        final String edgetopright = "Wall/wall_edge_top_right.png";
        final String edgeleft = "Wall/wall_edge_left.png";
        final String edgeright = "Wall/wall_edge_right.png";
```